pkgs/os-specific/linux/kernel/linux-testing-bcachefs.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchgit, perl, buildLinux, ... } @ args:
+
+import ./generic.nix (args // rec {
+  version = "4.11-git";
```

Comment:
because bcachefs has no stable release yet. they are working to get it merged upstream but until then i want to have this kernel package to test and help finding bugs.

>Bcachefs can currently be considered beta quality.

http://bcachefs.org/